_component_design/tooltip.md

```diff
@@ -13,28 +13,292 @@ last-modified: 2017-08-17
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Help text with a link
+{: .hxSubSectionTitle}
+
+Use this variation when a tooltip provides access to a singular external webpage.
+
+- This link should always be an external link and should open in a new tab. 
+- Be sure you always include an external link icon.
+- If you need additional inputs or interactions on a tooltip, for example, a dropdown or selector, use the larger popover pattern.
+
+See [Help links]({{site.baseurl}}/components/help-link.html) for link specifications.
+
+
+{% endcolumn %}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Spacing for tooltip with an associated link"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/help/tooltips/tooltip-text-with-link.svg){:width="100%"}
+
+{% endfigure %}
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Large tooltips
+{: .hxSubSectionTitle}
+
+Occassionally a tooltip needs to display a robust set of data that falls outside the bounds of traditional text conventions. The larger tooltips follow the same text conventions and sizing guidelines.
+
+{% endcolumn %}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Some tooltips contain larger amounts of information, like definitions and pricing structure. Such tooltips **should be used sparingly**."] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/help/tooltips/tooltip-large-tooltips.svg){:width="100%"}
+
 {% endfigure %}
 {% endcolumn %}
 </div>
+
+## Loading state
+{: .hxSectionTitle}
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+Loading states are rarely found when using the tooltip pattern. The loading state is applied exclusively when using data visualizations.
+
+{% endcolumn %}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Loading State."] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/help/tooltips/tooltip-loading.svg){:width="100%"}
+
+{% endfigure %}
+{% endcolumn %}
+</div>
+
+
+## Interaction behaviors
+{: .hxSectionTitle}
+
+Tooltips can be activated via either a hover or a click on the trigger element.
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Hover interaction
+{: .hxSubSectionTitle}
+
+
+Hover tooltips are best used on large breakpoints and desktop designs. Tooltips can have both hover and click interactions on the same tooltip across breakpoints. Hover enabled tooltips go through the following interaction lifecycle:
+ 
+- **Hidden:** The base state of a tooltip is to be hidden until the trigger is engaged by the pointed.
+- **Displayed:** Tooltips enter the display state once the pointer has hovered the trigger element for 500ms.
```

Comment:
missing `over` between `hovered` and `the`

Also, add a space between the number and the unit.

Reply:
dont.